lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -52,6 +52,8 @@ qx.Class.define("callbackery.ui.plugin.Action", {
                 case 'reload':
                 case 'cancel':
                 case 'wait':
+                case undefined:
+                    this.debug('Undefined action');
```

Comment:
this is the default case ... handled on line 57 for all undefined actions ... nur just the 'undefined action ...